components/builder-worker/src/runner/mod.rs

```diff
@@ -346,6 +382,13 @@ impl Runner {
     }
 }
 
+fn key_path() -> PathBuf {
+    (&*studio::STUDIO_HOME).lock().unwrap().join(format!(
```

Comment:
This is probably fine for now, especially if we clear it's contents at the end of a build. We want to make sure this isn't shared across builds though